### PR TITLE
SCRUM 4088 - Added Controller and Interface for Paralogy

### PR DESCRIPTION
**Pull request by VarunReddy1111**

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/jobs/executors/ParalogyExecutor.java [hunk 1]
```diff
@@ -94,5 +93,15 @@ private void runLoad(BulkLoadFileHistory history, String dataProvider, ParalogyI
 		ph.finishProcess();
 
 	}
+	
+	public APIResponse runLoad(String dataProvider, ParalogyIngestFmsDTO paralogyData) {
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/jobs/executors/ParalogyExecutor.java:97]: @VarunReddy1111 you shouldn't need this as the super class should be able to take care of it.

Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/jobs/executors/ParalogyExecutor.java:97]: We might not need this method as we can use the super class one.... maybe we need it due to having to deal with the "Pair" of genes.

Comment (VarunReddy1111) [reply src/main/java/org/alliancegenome/curation_api/jobs/executors/ParalogyExecutor.java:97]: Yes, I believe we need to have this method as we have overridden the base `runLoad` method in order to deal with "Pair" of genes.

Comment (VarunReddy1111) [reply src/main/java/org/alliancegenome/curation_api/jobs/executors/ParalogyExecutor.java:97]: I can exclude this for now, but for the following tickets, I would need this method as its used by the test cases.

Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/jobs/executors/ParalogyExecutor.java:97]: Ya... you are right sorry.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**